src/clarity-angular/wizard/providers/wizard-navigation.ts

```diff
@@ -143,7 +143,7 @@ export class WizardNavigationService implements OnDestroy {
 
     public setCurrentPage(page: WizardPage): void {
         this.currentPage = page;
-// TODO: emit page onload here
+        page.onLoad.emit(page.id);
```

Comment:
One thing here is the naming `onLoad`. How hard would it be to rename it to something related to what its doing - `makeCurrent` ? 

- I'm assuming `setCurrentPage` is called many times and to me I would think that `onLoad` is only called once.